### ISSUE TITLE
IP limit

### ISSUE THREAD

**Issue by nakulwastaken** [eichelb4rt/bh-rewards-bot#1]
brawlhalla's twitch reward extension limits 5 tracks per IP

**Comment by eichelb4rt** [comment]
Yeah that is a problem one could solve with proxies. I haven't implemented proxy support yet and i probably won't (unless i feel like it in the future).
